examples/tf/trpo_cartpole_bullet.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+"""This is an example to train a task with TRPO algorithm.
+
+Here it runs CartPoleBulletEnv environment with 100 iterations.
+
+Results:
+    AverageReturn: 100
+    RiseTime: itr 13
```

Comment:
These numbers were tested in `trpo_cartpole.py`. You need to update them (or delete them)